jupyterhub/apihandlers/users.py

```diff
@@ -18,6 +18,47 @@ def get(self):
         users = self.db.query(orm.User)
         data = [ self.user_model(u) for u in users ]
         self.write(json.dumps(data))
+    
+    @admin_only
+    @gen.coroutine
+    def post(self):
+        data = self.get_json_body()
+        if not data or not isinstance(data, dict) or not data.get('usernames'):
+            raise web.HTTPError(400, "Must specify at least one user to create")
+        
+        usernames = data.pop('usernames')
+        admin = data.get('admin', False)
```

Comment:
Ah, if admin is global, I'd add here your comment in the code itself (the one about no bulk creation of mixed admin/non-admin users).  Could be useful for anyone reading the code.